frontend/src/components/common/BackButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
can `QuestionSidebar` also use this?

Reply:
yes maaaam, but i refactored the `QuestionSidebar` in my other open PR (to get rid of the props + history in the file), so after I merge this one I was gonna make the update there 😅

Reply:
its fine if the refactor is not clean

Reply:
adding!